src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,35 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        //Excavator excavator = new Excavator();
+        //Bulldozer bulldozer = new Bulldozer();
+        //Truck truck = new Truck();
+
+        Object[] object = new Object[3];
+
+        object[0] = new Excavator();
+        object[1] = new Bulldozer();
+        object[2] = new Truck();
+
+
+        for (Object objects : object ) {
+            //Excavator
+            Excavator.doWork();
+            Excavator.stopWork();
+            //Bulldozer
+            Bulldozer.doWork();
+            Bulldozer.stopWork();
+            //Truck
+            Truck.doWork();
```

Comment:
Remove the explicit calls to 'Truck.doWork()' and 'Truck.stopWork()', as they should be invoked dynamically within the loop for each specific instance.